src/applications/vaos/utils/eligibility.js

```diff
@@ -73,31 +66,26 @@ export function getEligibilityChecks(
     requestLimitValue: null,
   };
 
-  if (!DIRECT_SCHEDULE_TYPES.has(typeOfCareId)) {
-    eligibility.directTypes = false;
-  } else {
-    if (
-      eligibilityData.directPastVisit.durationInMonths ===
-        DISABLED_LIMIT_VALUE ||
-      !eligibilityData.directPastVisit.hasVisitedInPastMonths
-    ) {
-      eligibility.directPastVisit = false;
-      eligibility.directPastVisitValue =
-        eligibilityData.directPastVisit.durationInMonths;
-    }
+  if (
+    eligibilityData.directPastVisit.durationInMonths === DISABLED_LIMIT_VALUE ||
```

Comment:
Minor but you could [decompose](https://refactoring.guru/decompose-conditional) some of the more complex conditionals with functions named for the check.